src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.State.cs

```diff
@@ -192,6 +192,14 @@ private TExpressionSyntax GetExpressionUnderSpan(SyntaxTree tree, TextSpan textS
                 }
 
                 var startToken = root.FindToken(textSpan.Start);
+                if (startToken.Span.End < textSpan.Start)
+                {
+                    // If we were pointing in the trailing trivia of a token, we shouldn't include that token
```

Comment:
i think we only want to do this if we're in whitespace-trivia.  We don't want to generally do this for all trivia before/after something.